yb-voyager/src/dbzm/yb_cdc_client.go

```diff
@@ -87,30 +89,24 @@ func (ybc *YugabyteDBCDCClient) GenerateAndStoreStreamID() (string, error) {
 	}
 	//stdout - CDC Stream ID: <stream_id>
 	streamID := strings.Trim(strings.Split(stdout, ":")[1], " \n")
-	//storing streamID in a file
-	streamIDFile := filepath.Join(ybc.exportDir, "metainfo", "yb_cdc_stream_id.txt")
-	file, err := os.Create(streamIDFile)
+	err = ybc.metaDB.UpdateMigrationStatusRecord(func(record *metadb.MigrationStatusRecord) {
+		record.YBCDCStreamID = streamID
+	})
 	if err != nil {
-		return "", fmt.Errorf("creating file: %s, error: %s", streamIDFile, err)
-	}
-	defer file.Close()
-	_, err = file.WriteString(streamID)
-	if err != nil {
-		return "", fmt.Errorf("writing to file: %s, error: %s", streamIDFile, err)
+		return "", fmt.Errorf("failed to update migration status record: %w", err)
```

Comment:
More specific error message: `failed to save the newly created stream ID %q in the msr: %w`